Behavioral changes for Data Prepper S3 sink (#4897)
opensearch-project/documentation-website: _data-prepper/pipelines/configuration/sinks/sinks.md

:--- | :--- |:------------| :---
routes | No | String list | A list of routes for which this sink applies. If not provided, this sink receives all events. See [conditional routing]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/pipelines#conditional-routing) for more information.
tags_target_key | No | String   | When specified, includes event tags in the output of the provided key.

> include_keys | No | String list | When specified, provides the keys in this list in the data sent to the sink. Note that some codecs or sinks may disallow this feature. See the documentation for your sink or codec.

Reviewer: ```suggestion
include_keys | No | String list | When specified, provides the keys in this list in the data sent to the sink. Some codecs do not allow use of this field. 
```

> exclude_keys | No | String list | When specified, excludes the keys given from the data sent to the sink. Note that some codecs or sinks may disallow this feature. See the documentation for your sink or codec.

Reviewer: ```suggestion
exclude_keys | No | String list | When specified, excludes the keys given from the data sent to the sink. Some codecs and sinks do not allow use of this field.
```
